bmi_visualization.py

```
@@ -9,7 +9,19 @@
 from collections.abc import Iterable
 from typing import Any, Protocol
 
-from bmi_core import auto_group, bmi_category, group_display_name
+# Import canonical BMI engine functions
+from core.bmi.engine import _auto_group, _bmi_category, _group_display_name, _normalize_bool_flag
+from core.i18n import Language, normalize_lang, t
```

Flake8 (7.3.0)

[error] 'core.i18n.Language' imported but unused

(F401)